tests/test_menu_launcher.py

```diff
@@ -270,6 +270,16 @@ def test_running_menu():
     # go to system commands
     child.sendline('5')
     child.expect('Return to Vent menu')
+    # choose reboot
+    child.sendline('4')
+    child.expect('Do you want to continue [Y/n]?')
```

Comment:
your tests might be happier if you make this slightly less specific.  curses on circleci is odd.